ansible/roles/load-balancer/defaults/main.yml

```diff
@@ -0,0 +1,6 @@
+keepalived_config_dir: /etc/keepalived/
+haproxy_config_dir: /etc/haproxy/
+
+keepalived_password: foobar
```

Comment:
This should also be exposed in group_vars/all so users can change. By default, keepalived_password in group vars would be commented-out.